.github/workflows/ci.yml

```diff
@@ -406,7 +403,6 @@ jobs:
   build_test_ubuntu:
     name: "Local: Build and unit testing on Ubuntu"
     runs-on: ubuntu-latest
-    needs: [smoke-tests]
     timeout-minutes: 35
```

Comment:
I understand you do this because you just want all of the different stages to run at the same time... but it is tipically a good practice to make dependent stages. That will reduce runner times consumption (even if your repo is public, do it for environmental reasons 😄 )

If the smoke tests fail.. the build tests are obviously going to fail. Just give it a thought do I understand your change.

Reply:
I see your point, but waiting 3-4 minutes to start the "real" testing is annoying. If we care about runner consumption, I would break the smoke test matrix in 4+4 or 2+2+2+2 .. but it seems not gain enough.

Reply:
Sure, yeah, I get it. No problem. It was just a suggestion 😄